packages/dds/task-manager/src/taskManager.ts

```diff
@@ -480,11 +460,19 @@ export class TaskManagerClass
 			return;
 		}
 
-		// If we're subscribed but not queued, we don't need to submit an abandon op (probably offline)
-		if (this.queued(taskId)) {
+		if (this.queuedOptimistically(taskId)) {
+			this.submitAbandonOp(taskId);
+			if (!this.assigned(taskId)) {
```

Comment:
It doesn't seem like this needs/should be conditionalized - we're abandoning so seems appropriate/truthful to emit "abandon" in all cases?  In particular thinking if this could be a problem if I volunteer and then abandon before I see the volunteer ack.

Reply:
Yeah, I think that makes sense and is also more in line with how `complete` works now. Updated in latest